ide/languages.hcl/nbproject/project.properties

```diff
@@ -14,5 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-javac.source=1.8
+javac.source=11
```

Comment:
This looks strange and wrong.

Reply:
I made this adjustment because my local build would not compile otherwise.  I'll take another look and revert if that works.